zipline/utils/calendars/us_futures_calendar.py

```diff
@@ -13,6 +13,12 @@
     Christmas
 )
 
+# Number of hours of offset between the open and close times dictated by this
+# calendar versus the 6:30am to 5:00pm times over which we want to simulate
+# futures algos.
+FUTURES_OPEN_TIME_OFFSET = 12.5
+FUTURES_CLOSE_TIME_OFFSET = -1
```

Comment:
Should I just put these numbers straight in the methods below instead of bothering to define them as constants?